pkg/jx/cmd/delete_app.go

```diff
@@ -120,58 +182,43 @@ func (o *DeleteAppOptions) Run() error {
 		}
 	}
 
-	if o.PullRequestPollTime != "" {
-		duration, err := time.ParseDuration(o.PullRequestPollTime)
-		if err != nil {
-			return fmt.Errorf("Invalid duration format %s for option --%s: %s", o.PullRequestPollTime, optionPullRequestPollTime, err)
-		}
-		o.PullRequestPollDuration = &duration
-	}
-	if o.Timeout != "" {
-		duration, err := time.ParseDuration(o.Timeout)
-		if err != nil {
-			return fmt.Errorf("Invalid duration format %s for option --%s: %s", o.Timeout, optionTimeout, err)
-		}
-		o.TimeoutDuration = &duration
-	}
-
 	if len(names) == 0 {
-		return fmt.Errorf("There are no Apps in Jenkins")
+		return deletedApplications, fmt.Errorf("There are no Apps in Jenkins")
```

Comment:
NIT: this should be applications as apps are a different thing.  (not your code)

Reply:
yeah, :-(  - it's littered with references to app rather than 'application'. This PR's not the space to address it :-/